### PR TITLE
Use appropriate labels for media nodes

### PR DESCRIPTION
**Pull request by ZoomTen**
Set imported Media node label to "Video Input" for video footage, and "Audio Input" for audio footage

This was prompted by troubleshooting a friend's node setup, and having unlabelled Media nodes made things kind of confusing (e.g. they would accidentally link a Volume node to a Media node containing only video)

### CODE REVIEW

**Comment by Simran-B** [comment]
Looks like Alex accidentally referenced the wrong issue (1422 instead of 1421). This PR looks good to me.

**Comment by ThomasWilshaw** [comment]
Might it be better to add a function to the media node that sets the label based on the stream type? That way if you add node through the UI and select say an audio stream the label becomes "Audio" as well?

**Comment by ZoomTen** [comment]
@ThomasWilshaw 

You have a point there, so I just pushed some additional commits that do just that, though my approach may be flawed

**Comment by ThomasWilshaw** [comment]
I think you should be able to simplify this a lot and keep it all in `media.cpp`. You shouldn't need the `HasUserLabel` method, just grab the current label in `FootageChanged` and compare it to what it should be autoamticaly. You'll probalably also need to call `FootageChanged` in `SetStream` . Finally you'll need to update `Retranslate` and add some logic there.

**Comment by itsmattkc** [comment]
Thanks for the PR, but I don't know if this exactly solves the problem. Before we merged the video and audio nodes, this is how they were named, and there was still confusion surrounding what exactly "Video Input" and "Audio Input" meant.

For example, if your node graph looked like "Video Input" -> "HelloWorld.MP4 (Clip)", people were confused about which node was the footage and which one was the clip (often they assumed the clip was the footage because it was labelled with the footage filename).

TL;DR, I think this is a much deeper issue than node labelling and requires a different approach.
